### PR TITLE
BROKERS: LoggingBroker-LogWarning

### LINKED ISSUE #30: BROKERS: LoggingBroker - LogWarning

### PR DESCRIPTION
**Pull request by SlimAhmad**
Closes #30

### CODE REVIEW

**Review by cjdutoit — CHANGES_REQUESTED** [review]
Resolve conflicts

**Review by cjdutoit — CHANGES_REQUESTED** [review]
Resolve build failure

File: GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs [hunk 1]
```diff
@@ -22,6 +22,11 @@ public class LoggingBroker : ILoggingBroker
 
         public void LogDebug(string message) =>
             this.logger.LogDebug(message);
+
+        public void LogWarning(string message) =>
+            this.logger.LogWarning(message);
+
+        public void LogCritical(Exception exception) =>
+             this.logger.LogCritical(exception, exception.Message);
```
Comment (cjdutoit) [inline GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs:30]: Remove this one.   We need [#31](https://github.com/The-Standard-Organization/GitFyle.Core.Api/issues/31) next and then LogCritical

Comment (SlimAhmad) [reply GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs:30]: @cjdutoit ok

**Review by cjdutoit — APPROVED** [review]
Resolve build failure

**Pull request merged**